### PR TITLE
@alloy => Can return artists by array of ids

### PR DESCRIPTION
**Pull request by eessex**
Adds `ids` param to `artists` schema, to return artists matching an array of passed ids.

### CODE REVIEW

**Comment by alloy** [comment]
Implementation looks good. The test could be a little improved, but otherwise 👍

File: src/schema/__tests__/artists.test.js [hunk 1]
```diff
@@ -26,4 +26,26 @@ describe("Artists", () => {
     const { artists } = await runQuery(query, { artistsLoader })
     expect(artists).toEqual([{ name: "Han Myung-Ok" }])
   })
+
+  it("returns a list of artists matching array of ids", async () => {
+    const artistsLoader = ({ ids }) => {
+      if (ids) {
+        return Promise.resolve([
+          {
+            name: "Han Myung-Ok",
```
[the remark this answers is not in the document]
Comment (eessex) [reply src/schema/__tests__/artists.test.js]: Makes sense to me!  Updated now.

File: src/schema/__tests__/artists.test.js [hunk 2]
```diff
@@ -26,4 +26,27 @@ describe("Artists", () => {
...
+      if (ids) {
+        return Promise.resolve([
+          {
+            _id: "52c721e5b202a3edf1000072",
+            name: "Han Myung-Ok",
+          },
+        ])
+      }
+      throw new Error("Unexpected invocation")
+    }
```
Comment (alloy) [inline src/schema/__tests__/artists.test.js]: I was actually thinking of _using_ the passed in `ids`, so something like:

```js
const artistsLoader = ({ ids }) => Promise.resolve(ids.map(_id => ({ _id })))
```

That way you can rest assured that the data you put in influences the data that comes out.

Comment (eessex) [reply src/schema/__tests__/artists.test.js]: Oh I see yes-- thanks for clarifying this test pattern

Comment (eessex) [reply src/schema/__tests__/artists.test.js]: This is updated now, do you have other changes before this is ready to merge?

**Comment by alloy** [comment]
Sorry for the delay, looking good!

**Pull request merged**